spec/acceptance/suites/default/10_yum_spec.rb

```diff
@@ -104,4 +56,29 @@
       end
     end
   end
+  context 'reset the yum repo back to normal' do
+    it 'should set up hiera' do
+      block_on(hosts, :run_in_parallel => true) do |host|
+        os = JSON.load(on(host,'puppet facts').stdout)['values']['os']
+        yum_updates_url = case "#{os['name']}-#{os['release']['full']}"
+          when /OracleLinux-7/
```

Comment:
Should be moved into the nodeset (unless we shouldn't have it present in some cases)

Reply:
This logic is for the `simp::yum` classes, so it's testing that capability. Also I forgot you could put yumrepo stuff in nodesets